src/Parameters.jl

```diff
@@ -548,7 +548,23 @@ function with_kw(typedef, mod::Module, withshow=true)
     else
         :nothing
     end
-    _pack = ismutable ? :_pack_mutable : :_pack_immutable
+    if ismutable
+        pack_macros = quote
+            macro $pack!_name(ex)
+                esc($Parameters._pack_mutable(ex, $unpack_vars))
+            end
+            macro $pack_name(ex)
+                Base.depwarn("The macro `@$($(Meta.quot(pack_name)))` is deprecated, use `@$($(Meta.quot(pack!_name)))`", $(QuoteNode(pack_name)) )
+                esc($Parameters._pack_mutable(ex, $unpack_vars))
```

Comment:
We should drop the deprecation (but I can do that).

Maybe it would be good to provide `@pack_T` for mutables as well, which would make a new instance?